ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.13.yaml

```
@@ -0,0 +1,110 @@
+base_images:
+  base:
+    name: "4.13"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.19:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+  os-min8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+images:
+- context_dir: .
+  dockerfile_path: cnf-tests/Dockerfile.openshift
+  from: base
+  inputs:
+    openshift_release_golang-1.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.19
+  to: cnf-tests
+- context_dir: tools/s2i-dpdk/
+  dockerfile_path: Dockerfile
+  from: base
+  to: dpdk
+- context_dir: .
+  dockerfile_path: ztp/resource-generator/Containerfile
+  from: os-min8
```

change base image to ubi9
```suggestion
  from: os-min9
```

---

same as above